components/atoms/Tables/TitleTickets.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export default function TitleTickets() {
+  return (
+    <table className="w-[778px] h-[28px]">
+      <thead className="w-full">
+        <tr className="w-full">
+          <td className="pl-[33px] w-1/5 font-normal not-italic text-[16px] leading-[28px] text-[#616E87]">
+            Номер заявки
+          </td>
+          <td className="w-2/12 font-normal not-italic text-[16px] leading-[28px] text-[#616E87] ">
+            Дата
+          </td>
+          <td className="w-3/12 font-normal not-italic text-[16px] leading-[28px] text-[#616E87]">
+            Тема обращения
+          </td>
+          <td className="w-2/12 font-normal not-italic text-[16px] leading-[28px] text-[#616E87]">
+            Описание
+          </td>
+          <td className="w-2/12 font-normal not-italic text-[16px] leading-[28px] text-[#616E87]">
+            Статус
+          </td>
+        </tr>
+      </thead>
+    </table>
+  );
+}
```

Comment:
```suggestion
export default function TitleTickets() {
  const tickets = [
    { title: 'Номер заявки', additionalClasses: 'pl-[33px] w-1/5' },
    { title: 'Дата', additionalClasses: 'w-2/12' },
    { title: 'Тема обращения', additionalClasses: 'w-3/12' },
    { title: 'Описание', additionalClasses: 'w-2/12' },
    { title: 'Статус', additionalClasses: 'w-2/12' }
  ]
  return (
    <table className="w-[778px] h-[28px]">
      <thead className="w-full">
        <tr className="w-full">
          {
            tickets.map(item => (
              <td key={item.title} className={`font-normal not-italic text-[16px] leading-[28px] text-[#616E87] ${item.additionalClasses}`}>
                {item.title}
              </td>
            ))
          }
        </tr>
      </thead>
    </table>
  );
}
```

и не указывай ширину таблицы